### ISSUE TITLE
opensuse (tealdeer) can't find cache after updating

### ISSUE THREAD

**Issue by tckosvic** [dbrgn/tealdeer#329]
Installed tldr as tealdeer from opensuse 15.4 software.  tldr runs upon call for cache update.  there are files in /home/username/.cache/tealdeer directory.  In a subdirectory named "tldr-main" there are10 or so subdirectories.  I assume this is the cache.

When running, e.g., tldr tar as an example, response is "cache not found; run  tldr --update.  That has already been run.
Need a way to have tldr find the cache it has updated.

thanks,      tom kosvic

**Comment by dbrgn** [comment]
What does `tldr --show-paths` output?

**Comment by tckosvic** [comment]
See below:
`tom@mydesktop:~> tldr --show-paths
Unknown flag: '--show-paths'

Usage:

    tldr [options] <command>...
    tldr [options]
tom@mydesktop:~> 
`
hope this helps,    tom kosvic

**Comment by tckosvic** [comment]
See below for 2nd response:
`tom@mydesktop:~> tldr --version
tealdeer v1.4.1 (Linux)
tom@mydesktop:~> 
`
thanks,    tom kosvic

**Comment by dbrgn** [comment]
Ah, that's an old version. The current version is 1.6.1.

Can you show the output of `tldr --config-path` in that case?

**Comment by tckosvic** [comment]
See below:  

tom@mydesktop:~> tldr --config-path
Config path is: /home/tom/.config/tealdeer/config.toml
tom@mydesktop:~>

**Comment by tckosvic** [comment]
See below again.  Dont know where strikeout came from:

`tom@mydesktop:~> tldr --config-path
Config path is: /home/tom/.config/tealdeer/config.toml
tom@mydesktop:~> `

**Comment by tckosvic** [comment]
Perhaps, I should compile it if the openSUSE packaged version is too out of date

**Comment by niklasmohrin** [comment]
Your old version still expects the name "master" for the pages from the repository, but the download gives you the new name "main". Updating should fix the issue :)

**Comment by niklasmohrin** [comment]
@tckosvic Did recompiling fix your issue? If so, we can close the issue
